anchor/common/qbft/src/lib.rs

```diff
@@ -832,14 +836,17 @@ where
             return;
         }
 
-        // Bump the current round
-        self.current_round = next_round;
-
-        // Set the state so SendRoundChange so we include Round + 1 in message
-        self.state = InstanceState::SentRoundChange;
-
-        self.send_round_change(Hash256::default());
-        self.start_round();
+        self.requested_round = next_round;
```

Comment:
`requested_round` is initially equal to `current_round` and this seems to be the only place where `requested_round` is updated. How is the value updated by this "request"?